src/runtime/hexagon/sim/driver/pthread.h

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Overall comment: do we need to use low level pthread api? could we use C++11 thread and port our TVM CPU thread pool implementation if we want to support Hexagon parallel? Does we have some special concern so that we have to use pthread api?

Reply:
The reason why this is here is to be able to run programs that use pthreads on the simulator, at least many of them.  The simulator does not support threads, there is only one execution thread at any time.  When you have a TVM schedule that uses `parallel`, it will use `std::thread`, which under the covers will call pthread functons.  We still want to be able to run it on the simulator (although with only 1 thread).